tests/simulation/service/test_translator_service.py

```diff
@@ -381,6 +381,7 @@ def test_simulation_to_all_translation():
                     entities=[Surface(name="wing1"), Surface(name="wing2")],
                 ),
                 Freestream(entities=[Surface(name="farfield")]),
+                Rotation(),
```

Comment:
This doesn't throw validation error?

Reply:
This is a debug code. Included by mistake